Source/Scene/Cesium3DTile.js

```diff
@@ -142,6 +142,8 @@ define([
         }
         this._viewerRequestVolume = viewerRequestVolume;
 
+        this._commandsLength = 0;
```

Comment:
This should have a readonly property getter.

Up to you, but this and the other "length" properties for rendering debugging output could be prefixed like `_debugCommandsLength`